api/machine_types.go

```diff
@@ -17,6 +17,7 @@ const (
 	MachineFlyPlatformVersion2                 = "v2"
 	MachineProcessGroupApp                     = "app"
 	MachineProcessGroupFlyAppReleaseCommand    = "fly_app_release_command"
+	MachineProcessGroupFlyAppEphemeralRunner   = "fly_app_ephemeral_runner"
```

Comment:
ephemeral_runner seems very generic, do you have more use cases in mind?
There was a discussion about showing the value of the process group name on the dashboard UI, having `fly_app_console` will read better than a generic `ephemeral runner`. may not..

Reply:
I deliberately left this open in case we end up implementing a `fly run`–type command in the future, since internally a few people seem enthusiastic about such a thing. (I sort of wonder if we’ll get any customer feedback along these lines once `fly console` is shipped.)

Would definitely be open to renaming now and creating another process group name if/when that happens, I was just concerned about how many special process group names we want to keep track of.

Reply:
Cool, thanks! Switched it to `fly_app_console` as suggested for now. If this ever grows into `fly run` with more process groups I think the namespace idea is a good call.